src/Backend/Synapse/ToStage/FromTableCTASAdapter.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\ImportExport\Backend\Synapse\ToStage;
+
+use Doctrine\DBAL\Connection;
+use Keboola\Db\ImportExport\Backend\CopyAdapterInterface;
+use Keboola\Db\ImportExport\Backend\Synapse\Exception\Assert;
+use Keboola\Db\ImportExport\Backend\Synapse\SynapseImportOptions;
+use Keboola\Db\ImportExport\ImportOptionsInterface;
+use Keboola\Db\ImportExport\Storage;
+use Keboola\Db\ImportExport\Storage\Synapse\SelectSource;
+use Keboola\Db\ImportExport\Storage\Synapse\Table;
+use Keboola\TableBackendUtils\Escaping\SynapseQuote;
+use Keboola\TableBackendUtils\Table\Synapse\TableIndexDefinition;
+use Keboola\TableBackendUtils\Table\SynapseTableDefinition;
+use Keboola\TableBackendUtils\Table\SynapseTableQueryBuilder;
+use Keboola\TableBackendUtils\Table\SynapseTableReflection;
+use Keboola\TableBackendUtils\Table\TableDefinitionInterface;
+use Keboola\TableBackendUtils\TableNotExistsReflectionException;
+
+class FromTableCTASAdapter implements CopyAdapterInterface
+{
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function runCopyCommand(
```

Comment:
imho by si to zaslouzilo sqlbuilder a par unit testu na tu query

Reply:
No původně sem si to mockoval jako celek aj sem to z 99% měl hotové, ale sestavit getColumnsDefinitions odpověď z db pro synapse je sebevražda :D možu to trochu zrefaktorovat to máš pravdu :+1:

Reply:
no myslel jsem to stejnym pristupem jako mame [SqlBuilder](https://github.com/keboola/php-db-import-export/blob/master/src/Backend/Synapse/ToFinalTable/SqlBuilder.php) v `ToFinalTable` . Ale koukam, ze ani tam to neni krasny jednoduchy unitest co jen assertuje string query... slo mi primarne o to, ze ty testy v tomto pripade hezky ukazuji jak ta query ma/bude vypadat, takze to ma i dost velky dokumentacni prinos treba pro me kdyz do toho musi bootnout